src/backend/src/controllers/reimbursement-requests.controllers.ts

```diff
@@ -50,6 +50,25 @@ export default class ReimbursementRequestsController {
     }
   }
 
+  static async reimburseUser(req: Request, res: Response, next: NextFunction) {
+    try {
+      const user = await getCurrentUser(res);
+      const { recipientId } = req.params;
+      const { amount } = req.body;
+
+      const recipientIdString = parseInt(recipientId);
+
+      if (isNaN(recipientIdString)) {
```

Comment:
remove lines 61-63 that's what's causing your error

Reply:
called it